Dockerfile

```diff
@@ -1,3 +1,5 @@
 FROM nginx:1.15-alpine
 COPY dist/hypertrace-ui /usr/share/nginx/html
-COPY conf/default.conf /etc/nginx/conf.d/default.conf
\ No newline at end of file
+COPY conf/default.conf /etc/nginx/conf.d/default.conf
+EXPOSE 2020
+HEALTHCHECK --interval=1s --retries=5 --timeout=1s CMD wget -qO- http://localhost:2020/graphql?query={__schema{queryType{name}}} &> /dev/null || exit 1
```

Comment:
does it need escaping the special characters like ?, =, {, } with `\`?

Reply:
no. I tested it and it's working! 
```
/ # time wget -qO- http://localhost:2020/graphql?query={__schema{queryType{name}}}
{"data":{"__schema":{"queryType":{"name":"Query"}}}}real	0m 0.00s
user	0m 0.00s
sys	0m 0.00s
```

```
/ # time wget -qO- http://localhost:2020/graphql?query={__schema{queryType{name}}}
wget: server returned error: HTTP/1.1 502 Bad Gateway
Command exited with non-zero status 1
real	0m 18.54s
user	0m 0.00s
sys	0m 0.00s
```